lms/djangoapps/verify_student/models.py

```diff
@@ -1103,13 +1104,13 @@ def invalidate_deadline_caches(sender, **kwargs):  # pylint: disable=unused-argu
     cache.delete(VerificationDeadline.ALL_DEADLINES_CACHE_KEY)
 
 
-class VerificationCheckpoint(models.Model):
-    """Represents a point at which a user is asked to re-verify his/her
-    identity.
-
-    Each checkpoint is uniquely identified by a
-    (course_id, checkpoint_location) tuple.
+class VerificationCheckpoint(DeprecatedModel):  # pylint: disable=model-missing-unicode
```

Comment:
If you made `DeprecatedModel` a mixin instead of a base class, maybe you wouldn't need the `__init__` here?  Actually, I don't understand why you need the `__init__` here at all.